### PR TITLE
User can run benchmarks with SecurityManager

### PR DESCRIPTION
**Pull request by prettyClouds**
Co-authored-by: Murtuza Boxwala <mboxwala@pivotal.io>
Co-authored-by: Kamilla Aslami <kaslami@pivotal.io>

### CODE REVIEW

**Comment by pivotal-jbarrett** [comment]
Looks like maybe some tests that span multiple PRs in here. That's ok.

File: geode-benchmarks/src/main/java/org/apache/geode/benchmark/topology/ClientServerTopology.java [hunk 1]
```diff
@@ -69,18 +70,26 @@ public static void configure(TestConfig testConfig) {
     }
 
     String withSslArg = System.getProperty("withSsl");
```
Comment (pivotal-jbarrett) [inline geode-benchmarks/src/main/java/org/apache/geode/benchmark/topology/ClientServerTopology.java]: Use `Boolean.getBoolean`

File: geode-benchmarks/src/main/java/org/apache/geode/benchmark/topology/ClientServerTopology.java [hunk 2]
```diff
@@ -69,18 +70,26 @@ public static void configure(TestConfig testConfig) {
...
-    if (withSslArg != null && withSslArg.equals("true")) {
-      logger.info("Configuring JVMs to run with SSL enabled");
-      testConfig.jvmArgs(CLIENT, Arrays.append(JVM_ARGS, WITH_SSL_ARGUMENT));
-      testConfig.jvmArgs(LOCATOR, Arrays.append(JVM_ARGS, WITH_SSL_ARGUMENT));
-      testConfig.jvmArgs(SERVER, Arrays.append(JVM_ARGS, WITH_SSL_ARGUMENT));
-    }
+    addToTestConfig(testConfig, withSslArg, WITH_SSL_ARGUMENT);
+
+    String withSecurityManagerArg = System.getProperty("withSecurityManager");
+    addToTestConfig(testConfig, withSecurityManagerArg, WITH_SECURITY_MANAGER_ARGUMENT);
```
Comment (pivotal-jbarrett) [inline geode-benchmarks/src/main/java/org/apache/geode/benchmark/topology/ClientServerTopology.java]: It turns out that `testConfig.jvmArgs` are additive so you don't need to merge them before calling this function.

File: geode-benchmarks/src/main/java/org/apache/geode/benchmark/topology/ClientServerTopology.java (hunk 1 above)
Comment (prettyClouds) [reply geode-benchmarks/src/main/java/org/apache/geode/benchmark/topology/ClientServerTopology.java]: cool method

File: geode-benchmarks/src/main/java/org/apache/geode/benchmark/topology/ClientServerTopology.java (hunk 2 above)
Comment (prettyClouds) [reply geode-benchmarks/src/main/java/org/apache/geode/benchmark/topology/ClientServerTopology.java]: good catch

**Pull request merged**